h2o-docs/src/booklets/v2_2015/source/GLMBooklet.tex

```diff
@@ -571,6 +571,120 @@ \subsubsection{Tweedie Models}
 \centering
 \includegraphics[scale=0.5]{../../../product/images/tweedie_deviance.png}
 \end{figure}
+
+
```

Comment:
Please put this somewhere in the booklet:

If maximum likelihood method is specified in dispersion_parameter_method for negative binomial, gamma and tweedie families, set parameter solve to "IRLSM".